protocol-units/execution/opt-executor/src/executor/execution.rs

```diff
@@ -41,51 +41,50 @@ impl Executor {
 			}
 		};
 
+		let block_executor = self.block_executor.clone();
```

Comment:
How large is the state that is being cloned here?

Reply:
It's an `Arc`.